### ISSUE TITLE
[Linux]Cannot open file "fsgame.ltx" Check your working folder

### ISSUE THREAD

**Issue by EluWriter** [Swartz27/xray-16#1553]
**Describe the bug**
Neither Clear Sky nor Call of Pripyat (STEAM versions in Russian language) start, displaying the following error message:

> Cannot open file "fsgame.ltx".
> Check your working folder

**To Reproduce**
1. Install OpenXRay
2. Start the game

**Expected behavior**
Games should start and not display any error

**Desktop (please complete the following information):**
 - OS: Debian 12
 - OpenXRay build version - 9065

**Additional context**
Around a month or two ago I played Call of Pripyat via OpenXRay without any issues. Since then I have reinstalled my OS. 
Tried to fix this bug by copying fsgame.ltx from /usr/share/openxray to /.local/share/GSC Game World/S.T.A.L.K.E.R. - Clear Sky and same for Call of Pripyat. The error disappeared but neither game started.

**Comment by Hrusteckiy** [comment]
Run the game from game root, not from binaries folder.

**Comment by Xottab-DUTY** [comment]
> The error disappeared but neither game started.

Can you look at the log, maybe there's something useful?

**Comment by EluWriter** [comment]
> Run the game from game root, not from binaries folder.

What do you mean? Can you tell me exactly what to do?

> Can you look at the log, maybe there's something useful?

Where can I find that log?
If I launch it from the terminal, this is what I get:

```elu@elu:~$ xr_3da -cs
! Couldn't create surface from image: Parameter 'src' is invalid
OpenXRay Release Master Gold build 9065, Dec  2 2023 (Linux 64-bit, shared)
Custom build from commit[] branch[]
 
command line -cs 

* CPU features: RDTSC, MMX, SSE, AVX
* CPU threads: 12

Initializing File System...
$fs_root$ = /home/elu/.local/share/GSC Game World/S.T.A.L.K.E.R. - Clear Sky/
using fs-ltx fsgame.ltx
----total count =  0```

**Comment by ghost** [comment]
I thought the game was necessary only for compiling, since I didn't scrolled down when downloading the precompiled build.
It is a mod, and it requires the original game. It isn't standalone yet.

**Comment by AMS21** [comment]
> I thought the game was necessary only for compiling, since I didn't scrolled down when downloading the precompiled build.

The original game is very much required since for legal reasons we can't ship the original gamedata, we only provide an updated engine + our own gamedata for shaders etc. Also strictly speaking you can compile OpenXRay without the original game you just can't play it.

**Comment by EluWriter** [comment]
I have the original game files, I have copied them per instructions where these files are needed.

**Comment by r-a-sattarov** [comment]
I have the same problem when running engine on Linux. I am using the version dated January 5, 2024.
Previously, I used the version dated October 31, 2023, everything works fine there.

**Comment by mgueji** [comment]
I have the same problem, I managed to run Clear Sky  version by creating a link to fsgame.ltx and the gamedata folder from `/usr/share/openxray` as the binary created a broken link to `"usr/share"/openxray/fsgame` and the other files by adding the quote marks to the path.

Anyways, when trying to run the game or the options it crashed again, is not finding some files, Iĺl try to work out what is wrong and update the post.

**Comment by carhub** [comment]
Windows 10, Steam-версия Чистого Неба, ловлю такую же ошибку, fsgame.ltx в корне игры.
![Безымянный](https://github.com/OpenXRay/xray-16/assets/64554549/dfed1b24-50ce-47b2-8c91-a05ed15e4e22)

**Comment by Hrusteckiy** [comment]
@carhub, а ты как запускаешь?

**Comment by carhub** [comment]
> @carhub, а ты как запускаешь?

Сначала пытался с папки bin, потом увидел, что надо запустить с корня игры, но не понял как.

**Comment by Hrusteckiy** [comment]
-_- запускай через ярлык, в котором рабочей папкой является корень игры. Это всегда и везде было на данном движке.
![image](https://github.com/OpenXRay/xray-16/assets/47980896/741a95dc-b564-4c1a-aa24-f892584f8c9f)

**Comment by r-a-sattarov** [comment]
As far as I understand, this behavior started from this https://github.com/OpenXRay/xray-16/commit/64778f161c02a95e3210153b22e069306922ee23 commit.

Also, I noticed that the Luabind library is now built as static (.a) rather than dynamic (.so).

**Comment by r-a-sattarov** [comment]
> Also, I noticed that the Luabind library is now built as static (.a) rather than dynamic (.so).

Using the `-DLUABIND_BUILD_SHARED=TRUE` option solves this problem, but there remains an error with the launch of the engine.

**Comment by r-a-sattarov** [comment]
Error with opening fsgame.ltx file was fixed with commit https://github.com/OpenXRay/xray-16/commit/6144d318e2e2f1030b767d6be422152b0d156e2b. But still need to use `-DLUABIND_BUILD_SHARED=TRUE` option.

**Comment by Xottab-DUTY** [comment]
Closing due to:
> Error with opening fsgame.ltx file was fixed with commit [6144d31](https://github.com/OpenXRay/xray-16/commit/6144d318e2e2f1030b767d6be422152b0d156e2b).
